docs/resources/azure_network_security_groups.md.erb

```diff
@@ -0,0 +1,65 @@
+---
+title: About the azure_network_security_groups Resource
+platform: azure
+---
+
+# azure\_network\_security\_groups
+
+Use the `azure_network_security_groups` InSpec audit resource to enumerate
+Network Security Groups.
+
+<br />
+
+## Syntax
+
+An `azure_network_security_groups` resource block identifies Network Security
+Groups by Resource Group.
+
+    describe azure_network_security_groups(resource_group: 'ExampleGroup') do
+      ...
+    end
+<br />
```

Comment:
Still don't know why we need these awkward HTML line break elements, but the pattern is to put an empty line above and below.